### DISCUSSION TITLE
debug1 ?

### DISCUSSION THREAD

**Discussion by wayne-shih** [rsms/inter#480] [Q&A] [answered]
Curious with the `debug1` glyph...

I realise this isn't marked to export so won't be displayed on the binary files. So, wondering how the `debug1` glyph gets used for debugging or testing and what each of the lines+positions mean for debugging 

<img width="150" alt="CleanShot 2022-08-27 at 16 27 23@2x" src="https://user-images.githubusercontent.com/51947763/187014305-c62b05be-b1bb-4b70-b15a-0522275078a0.png">

**Comment by rsms** [comment, marked as answer]
When working the font you can enable it to be exported and then use that for testing various shapers. It was particularly useful early on when Inter was designed specifically for 11dp, which at its old UPM was what the various lines and blocks in that glyph measured. I haven't used it in years though so can just remove it.

**Reply by rsms** [reply]
> What size is Inter targeting now? And would the new UPM change fix/change #166 in any way?

Nope, at least it's not intended to change the appearance. As [Stephen points out](https://github.com/rsms/inter/issues/166#issuecomment-499595660), what you are seeing is most likely the result from [Microsoft ClearType](https://learn.microsoft.com/en-us/typography/cleartype/). You can customize ClearType in Windows >=10 with "Adjust ClearType text"

**Reply by rsms** [reply]
> What size is Inter targeting now?

All rasterization sizes, or: no size in particular. With a 2048 UPM, 128 units of a glyph corresponds to exactly one pixel at rasterization size 16px.
